prowler/providers/aws/lib/arguments/arguments.py

```diff
@@ -196,7 +196,7 @@ def validate_arguments(arguments: Namespace) -> tuple[bool, str]:
     if (
         (arguments.session_duration and arguments.session_duration != 3600)
         or arguments.external_id
-        or arguments.role_session_name
+        or arguments.role_session_name != "ProwlerAssessmentSession"
```

Comment:
Please store `ProwlerAssessmentSession` as a constant and use it in the code (search for other occurrences).